src/ostorlab/cli/ci_scan/run/assets/android_aab.py

```diff
@@ -0,0 +1,44 @@
+"""Asset of type .APK package file.
+This module takes care of preparing a file of type .APK before calling the create mobile scan API.
+"""
+import io
+import click
+from ostorlab.cli.ci_scan.run import run
+from ostorlab.apis.runners import authenticated_runner
+from ostorlab.apis.runners import runner as base_runner
+from ostorlab.apis import scan_create as scan_create_api
+
+
+@run.run.command()
+@click.argument('file', type=click.File(mode='rb'), required=True)
+@click.pass_context
+def android_aab(ctx: click.core.Context, file: io.FileIO) -> None:
+    """Create scan for android .APK package file."""
+    ci_logger = ctx.obj['ci_logger']
```

Comment:
This has a lot of duplicate code between all the assets. I would suggest putting it in a method and use across the assets.